### ISSUE TITLE
Django Basic Permission Spike

### ISSUE THREAD

**Issue by amymok** [HHS/OPRE-OPS#142]
### Goals
* Better understand Django basic permission functionalities and create tasks for #141 

### Tasks
- [ ] Research Django basic permission for admin interface (including types of users/groups)
- [ ] Provide recommendation on what to implement for #141

### Additional Context
* #141

### Resources
* https://docs.djangoproject.com/en/3.2/ref/contrib/auth/#

**Comment by alexsoble** [comment]
I think we said I'd pick this one up, right? 😄

**Comment by amymok** [comment]
Yes, I assigned it to you!  And you are going to pair with Carly on this.
